packages/oui-action-menu/src/action-menu-item.directive.js

```diff
@@ -0,0 +1,54 @@
+import controller from './action-menu-item.controller'
+
+const baseClass = 'oui-action-menu__item'
+const dividerClass = 'oui-action-menu__item_divider'
+const buttonClasses = 'oui-button oui-button_action-menu'
+
+const linkTemplate = (dividerClass, external) => `
+  <li class="${baseClass} ${dividerClass}">
+    <a
+      class="${buttonClasses}"
```

Comment:
I don't want to make a binding, I want to avoid watch. Yes, I can make a `::`. You don't see it here but in test context (I don't know why), ng-class with `{ 'xxx': ::booleanVar }` are not correctly parsed and throw errors.
So, instead of making a binding in one place and interpolation in another place, I make interpolation everywhere here.